doc/openocd.texi

```diff
@@ -10753,6 +10753,19 @@ For details on this trigger type, see the RISC-V Debug Specification.
 Clear the type 4 trigger that was set using @command{riscv itrigger set}.
 @end deffn
 
+@subsection RISC-V Program Buffer Commands
+
+Program Buffer is an optional feature of RISC-V targets - it is a mechanism that debuggers
+can use to execute sequences of arbitrary instructions (small programs) on the target.
+For details on the Program Buffer, please refer to the RISC-V Debug Specification.
+
+@deffn {Command} {riscv exec_progbuf inst1 [inst2 [... inst_n]]}
```

Comment:
Is it worth describing how compressed instructions fit into the 32-bit words?

Reply:
I believe that would be useful - I have added that information along with examples. I have made minor reword of the documentation, too.